src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServer1.java

```diff
@@ -112,8 +115,11 @@ public void run(RunConfiguration config) throws AppEngineException {
     for (File service : config.getServices()) {
       arguments.add(service.toPath().toString());
     }
+
+    Map<String, String> appEngineEnvironment = getAllAppEngineEnvironment(config.getServices());
```

Comment:
would it makes sense to perform this logic in the `runDevAppServer1Command` implementation and not pass it as a parameter? It seems like we would always do it.